### PR TITLE
Add database schema and queries

### PR DESCRIPTION
**Pull request by csaba-nagy**
### Description

The database schema and queries for CRUD operations have been added.

### Linked Issues
#2

### CODE REVIEW

**Review by nandordudas — APPROVED** [review]
hey, it's looking good!

just a note to formatting

```sql
select
  table.field
from
  table
  left join table2 on 1=1
    and table.field = table2.field
where 1=1
  and table.field = ...
```

**Comment by csaba-nagy** [comment]
I have changed the code format of the queries according to your suggestion.

**Review by nandordudas — APPROVED** [review]
pr can be merged

**Comment by nandordudas** [comment]
> hey, it's looking good!
> 
> just a note to formatting
> 
> ```sql
> select
>   table.field
> from
>   table
>   left join table2 on 1=1
>     and table.field = table2.field
> where 1=1
>   and table.field = ...
> ```

**Code change isn't required! Sorry, I didn't provide precise note before.**

tl;dr

use consitent format

- always use uppercase sql commands (select, from, join, ...)
- use indentation as relation connection helper
- trailing `on 1=1` and `where 1=1` should make more readable relations and criterieas
- table name prefixes are recommended of query contains more than one table
